packages/pg/lib/connection-parameters.js

```diff
@@ -112,6 +131,16 @@ class ConnectionParameters {
     if(this.ybServersRefreshInterval<0 || this.ybServersRefreshInterval>600){
       this.ybServersRefreshInterval = 300
     }
+    if (typeof this.fallbackToTopologyKeysOnly === 'string') {
+      this.fallbackToTopologyKeysOnly = this.fallbackToTopologyKeysOnly === 'true'
+    }
+    this.failedHostReconnectDelaySecs = Number(this.failedHostReconnectDelaySecs)
+    if(isNaN(this.failedHostReconnectDelaySecs) || !Number.isInteger(this.failedHostReconnectDelaySecs)){
+      throw new Error(' You need to Enter valid failedHostReconnectDelaySecs')
+    }
+    if(this.failedHostReconnectDelaySecs<0 || this.failedHostReconnectDelaySecs>60){
```

Comment:
```suggestion
    if (this.failedHostReconnectDelaySecs < 0 || this.failedHostReconnectDelaySecs > 60) {
```
Please check overall formatting of the modified and newly added code.